src/main/java/net/logstash/logback/stacktrace/ShortenedThrowableConverter.java

```diff
@@ -159,6 +168,15 @@
     @Override
     public void start() {
         parseOptions();
+        // instantiate stack hasher if need be
+        if(inlineHash) {
+            // use same exclusion patterns to compute hash
+            if(excludes == null || excludes.isEmpty()) {
+                stackHasher = new StackHasher();
+            } else {
+                stackHasher = new StackHasher(StackElementFilter.byPattern(excludes));
```

Comment:
Well, that's a very good question related to rule 3 (see below) that we could rephrase as:

"Does the exclusion patterns for outputing the stack trace match the exclusion patterns for computing the error hashes ?"

Not 100% sure about the answer, but in general case I would say **yes**.

What do you think ?